io/zenoh-transport/tests/unicast_transport.rs

```diff
@@ -441,14 +477,10 @@ async fn test_transport(
         "Sending {} messages... {:?} {}",
         MSG_COUNT, channel, msg_size
     );
-    let cctrl = match channel.reliability {
-        Reliability::Reliable => CongestionControl::Block,
-        Reliability::BestEffort => CongestionControl::Drop,
-    };
     // Create the message to send
     let message: NetworkMessage = Push {
         wire_expr: "test".into(),
-        ext_qos: QoSType::new(channel.priority, cctrl, false),
+        ext_qos: QoSType::new(channel.priority, CongestionControl::Block, false),
```

Comment:
Revert this change.

Reply:
Sometimes I get congestion, and with `CongestionControl::Drop` test fails here:
`client_transport.schedule(message.clone()).unwrap(); (line 499 in this branch)`

The original code sets `CongestionControl::Drop` for unreliable transports, BUT the congestion policy is applied at `pipeline.rs` input, which causes `Transport::schedule` to return `false` in case of congestion......
As far as we need to test the transport's message scheduling, we couldn't ignore `Transport::schedule` return value.